pkgs/top-level/aliases.nix

```diff
@@ -853,6 +853,7 @@ mapAliases ({
   solr_8 = solr; # added 2021-01-30
   spectral = neochat; # added 2020-12-27
   sundials_3 = throw "sundials_3 was removed in 2020-02. outdated and no longer needed";
+  sxiv = nsxiv; # added 2021-10-17
```

Comment:
I disagree with this change, you can't replace `sxiv` with `nsxiv` transparently if the new package doesn't provide the same binary name.

People will install `sxiv`, the installation process will work but user won't find `sxiv`, this is very confusing for users.